src/main/frontend/client/components/BerryTile.js

```diff
@@ -1,11 +1,12 @@
 import React from "react"
+import { Link } from "react-router-dom"
 
 const BerryTile = props => {
-  const { name, imgUrl, description } = props.berry
+  const { name, imgUrl, description, id } = props.berry
```

Comment:
We could even refactor this by destructuring in the parameter above!
For example:

```js
const BerryTile =({ name, imgUrl, description, id }) => {
```

Nothing you have to do, but just a potential refactor I wanted to point out.